README.md

```diff
@@ -103,7 +103,7 @@ You can also include it in a `requirements.yml` file along with other required c
 ---
 collections:
   - name: community.zabbix
-    version: 2.0.0
+    version: 2.0.1
```

Comment:
Please remove the change here.  We'll take care of it when we release an actual update.